elements/pl-multiple-choice/pl-multiple-choice.css

```diff
@@ -7,3 +7,7 @@
     min-width: 1.25em;
     overflow-wrap: normal;
 }
+
+.pl-multiple-choice-hint {
+    border-left: 4px solid #999;
```

Comment:
I wonder if it may be better to move all styling for this hint to the CSS (i.e., replace classes `mr-1 ml-1 pl-2` in the hint div with `margin-right`, `margin-left` and `padding-left` properties here). This could help with for consistency and make it easier to change in the future.

Also, I think it may be a good idea to introduce a light background or very thin border-top/bottom/right to distinguish it from the question itself.

Reply:
Will do. I added a light gray background to the hint, and a border radius so the corners don't look sharp. I'll commit the changes shortly.
<img width="854" alt="Screen Shot 2021-08-12 at 1 58 21 PM" src="https://user-images.githubusercontent.com/68084820/129269054-929cf2ca-68fc-45b3-a03a-06f669268c6d.png">